samples/wifi/softap/src/wpas_events.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/net/net_event.h>
+#include <ctrl_iface_zephyr.h>
+#include <supp_events.h>
+#include <supp_main.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+LOG_MODULE_REGISTER(wpas_event, CONFIG_LOG_DEFAULT_LEVEL);
+
+/* TODO: Handle other events */
```

Comment:
We cannot just keep copying TODOs :), as this is the second copy, now is the time to implement them.

Reply:
I would agree to remove this comment. If there is a task to add further events, handle the same in that PR

Reply:
Will create a task for this and handle in different PR.